server/app.es6

```diff
@@ -167,22 +170,9 @@ let createNoncommercialTempApp = (req, res) => {
       noncommercialFieldsLocationDescription: req.body.noncommercialFields.locationDescription,
       noncommercialFieldsStartDateTime: req.body.noncommercialFields.startDateTime,
       noncommercialFieldsEndDateTime: req.body.noncommercialFields.endDateTime,
-      noncommercialFieldsStartMonth: req.body.noncommercialFields.startMonth,
-      noncommercialFieldsStartDay: req.body.noncommercialFields.startDay,
-      noncommercialFieldsStartYear: req.body.noncommercialFields.startYear,
-      noncommercialFieldsEndMonth: req.body.noncommercialFields.endMonth,
-      noncommercialFieldsEndDay: req.body.noncommercialFields.endDay,
-      noncommercialFieldsEndYear: req.body.noncommercialFields.endYear,
-      noncommercialFieldsStartHour: req.body.noncommercialFields.startHour,
-      noncommercialFieldsStartMinutes: req.body.noncommercialFields.startMinutes,
-      noncommercialFieldsStartPeriod: req.body.noncommercialFields.startPeriod,
-      noncommercialFieldsEndHour: req.body.noncommercialFields.endHour,
-      noncommercialFieldsEndMinutes: req.body.noncommercialFields.endMinutes,
-      noncommercialFieldsEndPeriod: req.body.noncommercialFields.endPeriod,
       noncommercialFieldsNumberParticipants: req.body.noncommercialFields.numberParticipants,
       noncommercialFieldsSpectatorCount: req.body.noncommercialFields.spectators,
-      signature: req.body.signature,
-      reasonForReturn: req.body.reasonForReturn
```

Comment:
do we want to keep this in here (reasonForReturn) since it will be in the front end at some point?